http/http/src/main/java/io/helidon/http/ServerResponseHeadersImpl.java

```diff
@@ -63,15 +77,13 @@ public ServerResponseHeaders clearCookie(String name) {
                 if (!found) {
                     String[] values = new String[newValues.length + 1];
                     System.arraycopy(newValues, 0, values, 0, newValues.length);
-                    values[values.length - 1] = expiredCookie.text();
+                    values[values.length - 1] = expiredCookie.text();   // replace with expired
```

Comment:
IIUC the logic in the "then" part of this `if` handles the case where the cookie requested to be cleared _was not_ present in the original cookie list and so the expired version of the cookie is being _added_ to the end of a newly-created list that has an extra slot to hold the extra (expired) cookie. 

The comment using "replace" here confused me for a moment I guess.

Reply:
Will fix comment